docs/installation/server-config/config-reference.rst

```diff
@@ -158,6 +158,15 @@ By default, the service ID of KSQL servers is ``default_``. The service ID is al
 the prefix for the internal topics created by KSQL. Using the default value ``ksql.service.id``, the KSQL internal topics
 will be prefixed as ``_confluent-ksql-default_`` (e.g. ``_command_topic`` becomes ``_confluent-ksql-default__command_topic``).
 
+.. _ksql-internal-topic-replicas:
+
+--------------------
+ksql.internal.topic.replicas
+--------------------
+
+The number of replicas for the internal topics created by KSQL server. The default is one.
+This config parameter works in KSQL 5.3 and later.
+
 .. _ksql-sink-partitions:
 
```

Comment:
We should update the configs in the "Recommended KSQL Production Settings" section of this doc to remove `ksql.sink.replicas` since it's now deprecated, and replace it with the new config instead. (Also update the associated comment while we're at it.)

Reply:
Good point! I missed that one. Updated it.